Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.10-bookworm
```

Comment:
why not just `python:3.10` ?

Reply:
maybe another alternative is nvcr.io/nvidia/pytorch:24.12-py3 which torch and cuda embeded.